addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -168,4 +175,16 @@
 		<description>Media type of the current file</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="thumbnail" advanced="true">
+		<item-type>String</item-type>
```

Comment:
@kaikreuzer Unfortunately, the current implementation is not sufficient for Kodi, I opened an [issue](https://github.com/eclipse/smarthome/issues/3673). If I correct the MIME info, it works like a charm.